rts/lra/test/README.adoc

```diff
@@ -1,57 +1,63 @@
 = `lra/test` module - integration tests
 
+This module has been designed to simulate an environment with three actors: lra-coordinator, lra-participant,
+and a client (which in this case is the test class). This is implemented using three JVMs running separately.
+This reproduces a production scenario where lra-coordinator runs independently from the lra-participant(s)
+and from the client(s).
+All tests are executed with help of https://arquillian.org[Arquillian] platform. In some tests, Arquillian is coupled
+with maven plugin to start the runtime and deploy lra-coordinator.
+
 The `lra/test` module contains the following submodules
 
-* link:./arquillian-extension/[`arquillian-extension`] contains set of
-  Arquillian extension classes which makes possible to execute the test modules
-  on supported runtimes
+* link:./arquillian-extension/[`arquillian-extension`] contains Arquillian extension classes
+  which make possible to execute the test modules on supported runtimes
 * link:./basic[`basic`] contains integration tests which are written for testing
   the Narayana features of the
   https://github.com/eclipse/microprofile-lra[LRA specification]
-* link:./tck[`tck`] is a module which is responsible for execution of
+* link:./tck[`tck`] is a module which is responsible for the execution of
   https://github.com/eclipse/microprofile-lra/tree/master/tck[LRA TCK tests]
   with Narayana supported runtimes
-
-All tests are executed with help of https://arquillian.org[Arquillian] platform.
-Arquillian is responsible to launch the runtime and makes possible to execute
-the tests.
+* link:./crash[`crash`] hosts tests that simulate a crash of either lra-coordinator or
+  lra-participant or both. So far, only Wildfly is considered in this module as Quarkus
+  does not have an Arquillian adapter to run unmanaged tests
 
 == How to run integration tests
 
-The itegration tests which should be run on the application runtime
-are configured to be started with https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin].
+Integration tests are tested with
+https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin], using the target 'verify'.
 
-To run the tests the target `verify` has to be used.
-Single test with `-Dit.test=...`.
-Debug the Maven test execution with `-Dmaven.failsafe.debug`.
+Single test can be executed with `-Dit.test=...`
+To use Failsafe in debug mode, use `-Dmaven.failsafe.debug`
 
-The Failsafe Plugin link:../pom.xml#114[is configured] to run tests with names
+The Failsafe Plugin link:../pom.xml[is configured] to run tests with names
 `*IT.java`, `*Tests.java` and `*TestCase.java`.
 
 === Application runtimes
 
 The module is configured to run two runtimes - https://www.wildfly.org[WildFly]
-and link:https://quarkus.io[Quarkus]/link:https://thorntail.io[Thorntail].
+and link:https://quarkus.io[Quarkus].
```

Comment:
What about thorntail?

Reply:
Thorntail is not used to host lra-coordinator. At the moment, Thorntail is only used to host lra-participants. Also, in a next PR, Thorntail will be removed completely

Reply:
Which module is being referred to? Thorntail is still one of the `=== Application runtimes` even though it used to host a different application?

Reply:
I will restore the reference to Thorntail then. Thanks Michael :-)

Reply:
Thanks